library/tiqr/Tiqr/Service.php

```diff
@@ -346,11 +346,10 @@ public function sendAuthNotification(string $sessionKey, string $notificationTyp
             switch ($notificationType) {
                 case 'APNS':
                 case 'APNS_DIRECT':
-                    $apns_version = $this->_options['apns.version'] ?? 1;
-                    if ($apns_version ==2 )
-                        $message = new Tiqr_Message_APNS2($this->_options, $this->logger);
-                    else
-                        $message = new Tiqr_Message_APNS($this->_options, $this->logger);
+                    $apns_version = $this->_options['apns.version'] ?? 2;
+                    if ($apns_version !=2)
+                        throw new InvalidArgumentException("Unsupported APNS version '$apns_version'");
+                    $message = new Tiqr_Message_APNS2($this->_options, $this->logger);
```

Comment:
:wastebasket:   `Tiqr_Message_APNS` ?

Or is the class used elsewhere?

Reply:
Tiqr_Message_APNS implemented the Apple v1 APNS API that used the Zend library. This was indeed removed.